plasmapy/diagnostics/charged_particle_radiography/tests/test_synthetic_radiography.py

```diff
@@ -624,7 +614,7 @@ def test_saving_output(tmp_path):
     results_1 = sim.results_dict
 
     # Save result
-    path = str(tmp_path / "temp.npz")
+    path = os.path.join(tmp_path, "temp.npz")
```

Comment:
```suggestion
    path = tmp_path / "temp.npz"
```
The `tmp_path` fixture returns a `pathlib.Path` object, so it should be possible to do this instead of needing `os.path`.